docs/content/commands.md

```diff
@@ -182,43 +190,37 @@ public class BuildWallResponseHandler : ComponentSystem
+        * `TimeoutMillis` is optional. 
+    * Receiving a response - `WorldCommands.ReserveEntityIds.CommandResponses`. This contains a list of `WorldCommands.ReserveEntityIds.ReceivedResponse` structs. 
+* CreateEntity
+    * Sending a request - `WorldCommands.CreateEntity.CommandSender`. This contains a list of `WorldCommands.CreateEntity.Request` structs. Add a struct to the list to send the command.
+        * `EntityId` and `TimeoutMillis` are optional. 
+        * If you do specify an `EntityId`, you need to get this from a `ReserveEntityIds` command.
+    * Receiving a response - `WorldCommands.CreateEntity.CommandResponses`. This contains a list of `WorldCommands.CreateEntity.ReceivedResponse`. 
+* DeleteEntity
+    * Sending a request - `WorldCommands.DeleteEntity.CommandSender`. This contains a list of `WorldCommands.DeleteEntity.Request` structs. Add a struct to the list to send the command.
+        * `TimeoutMillis` is optional.
+    * Receiving a response - `WorldCommands.DeleteEntity.CommandResponses`. This contains a list of `WorldCommands.DeleteEntity.ReceivedResponse`.
+* EntityQuery
+    * Sending a request - `WorldCommands.EntityQuery.CommandSender`. This contains a list of `WorldCommands.EntityQuery.Request` structs. Add a struct to the list to send the command
+        * For more information, see [entity queries](https://docs.improbable.io/reference/latest/shared/glossary#queries) in the SpatialOS documentation.
+        * `TimeoutMillis` is optional.
+    * Receiving a response - `WorldCommands.EntityQuery.CommandResponses`. This contains a list of `WorldCommands.EntityQuery.ReceivedResponse`.
 
-        `timeoutMillis` is optional. 
-    * Response - `ReserveEntityIdsResponse`
-* CreateEntityRequest
-    * Method - `SendCreateEntityRequest(Worker.Entity entity, long entityId = 0, uint timeoutMillis = 0)`
 
-        `entityId` and `timeoutMillis` are optional.
-
-        If you do specify an `entityId`, you need to get this from `SendReserveEntityIdsRequest`.
-    * Response - `CreateEntityResponse`
-* DeleteEntityRequest
-    * Method - `SendDeleteEntityRequest(long entityId, uint timeoutMillis = 0)` 
-
-        `timeoutMillis` is optional. 
-    * Response - `DeleteEntityResponse`
-* EntityQueryRequest
-    * Method - `SendEntityQueryRequest(Worker.Query.EntityQuery entityQuery, uint timeoutMillis = 0)`
-
-        For more information, see [entity queries](https://docs.improbable.io/reference/13.0/shared/glossary#queries) in the SpatialOS documentation.
-
-        `timeoutMillis` is optional. 
-    * Response - `EntityQueryResponse`
-
-When a response is received, the Unity GDK attaches a `CommandResponses<T>` ECS component to the ECS entity that originally sent the request. `T` is the corresponding response type.
+When a response is received, the Unity GDK attaches the corresponding responses ECS component to the ECS entity that originally sent the request. 
```

Comment:
"When a response is received, the Unity GDK attaches the corresponding responses ECS component to the ECS entity that originally sent the request. "
This is a bit confusing. I think it should be:
"When a response is received, the Unity GDK attaches an ECS component to the ECS entity that originally sent the request. The ECS component it attaches corresponds to the response it receives."